setup.cfg

```diff
@@ -9,8 +9,5 @@ rednose=1
 [pep8]
 exclude=src/,.*
```

Comment:
With the pycodestyle section in tox.ini, do we still need this here?

Reply:
Nope, will remove.